save-backend/src/main/kotlin/com/saveourtool/save/backend/security/TestSuitePermissionEvaluator.kt

```diff
@@ -1,25 +1,44 @@
 package com.saveourtool.save.backend.security
 
+import com.saveourtool.save.backend.service.LnkOrganizationTestSuiteService
+import com.saveourtool.save.domain.Role
+import com.saveourtool.save.entities.Organization
 import com.saveourtool.save.entities.TestSuite
+import com.saveourtool.save.permission.Permission
+import com.saveourtool.save.permission.Rights
+import org.springframework.beans.factory.annotation.Autowired
 import org.springframework.security.core.Authentication
 import org.springframework.stereotype.Component
 
 /**
- * Class that is capable of assessing user's permissions regarding test suites.
+ * Class that is capable of assessing organization's permissions regarding test suites.
  */
 @Component
 class TestSuitePermissionEvaluator {
+    @Autowired
+    private lateinit var lnkOrganizationTestSuiteService: LnkOrganizationTestSuiteService
```

Comment:
Please prefer constructor injection over field injection

Reply:
Will fix it

Reply:
Made service be a property